frontend/src/app/pages/exercises/exercise/shared/simulation/signaller-modal/signaller-modal-region-commands/signaller-modal-region-commands.component.ts

```diff
@@ -0,0 +1,222 @@
+import { map, type Observable } from 'rxjs';
+import {
+    createSelectBehaviorStatesByType,
+    selectTransferPoints,
+} from 'src/app/state/application/selectors/exercise.selectors';
+import { SignallerModalDetailsService } from '../details-modal/signaller-modal-details.service';
+import type { InterfaceSignallerInteraction } from '../signaller-modal-interactions/signaller-modal-interactions.component';
+
+@Component({
+    selector: 'app-signaller-modal-region-commands',
+    templateUrl: './signaller-modal-region-commands.component.html',
+    styleUrls: ['./signaller-modal-region-commands.component.scss'],
+    standalone: false,
+})
+export class SignallerModalRegionCommandsComponent implements OnChanges {
+    @Input()
+    simulatedRegionId!: UUID;
+
+    @ViewChild('transferConnectionsEditor')
+    transferConnectionsEditor!: TemplateRef<any>;
+    @ViewChild('transferTraysEditor')
+    transferTraysEditor!: TemplateRef<any>;
+    @ViewChild('transportOfCategoryEditor')
+    transportOfCategoryEditor!: TemplateRef<any>;
+    @ViewChild('provideVehiclesEditor')
+    provideVehiclesEditor!: TemplateRef<any>;
+    @ViewChild('requestTargetEditor')
+    requestTargetEditor!: TemplateRef<any>;
+
+    ownTransferPointId$!: Observable<UUID>;
+    manageTransportBehaviorId$!: Observable<UUID | null>;
+    transferVehiclesBehaviorId$!: Observable<UUID | null>;
+    requestBehaviorId$!: Observable<UUID | null>;
+
+    commandInteractions: InterfaceSignallerInteraction[] = [
+        {
+            key: 'editTransferConnections',
+            name: 'Standort eines Bereichs',
+            details:
+                '(macht den Standort eines anderen Einsatzabschnitts bekannt)',
+            keywords: [
+                'bereich',
+                'bereiche',
+                'abschnitt',
+                'abschnitte',
+                'ort',
+                'lage',
+                'verknüpfung',
+                'verbindung',
+                'transfer',
+                'transport',
+                'kennen',
+                'bekannt',
+            ],
+            hotkey: new Hotkey('1', true, () => this.editTransferConnections()),
+            requiredBehaviors: [],
+        },
+        {
+            key: 'editTransferPatientTrays',
+            name: 'PAs für Abtransport festlegen',
+            details:
+                '(aus welchen Patientenablagen sollen Patienten ins Krankenhaus gebracht werden)',
+            keywords: [
+                'ablage',
+                'ablagen',
+                'patientenablage',
+                'patientenablagen',
+                'transport',
+                'transfer',
+                'krankenhaus',
+                'auswahl',
+                'abtransport',
+            ],
+            hotkey: new Hotkey('2', true, () =>
+                this.editTransferPatientTrays()
+            ),
+            requiredBehaviors: ['managePatientTransportToHospitalBehavior'],
```

Comment:
It's a little weird that these `requiredBehaviors` are defined twice: Once here, and again in the `*ngIf` of the respective `ng-template`.

Reply:
No, I don't have a great idea, either. I've thought about it a bit last week and couldn't come up with one, therefore my vague comment above. I'd say to just keep it like it is now.